go.mod

```diff
@@ -7,7 +7,9 @@ require (
 	github.com/Azure/azure-storage-blob-go v0.8.0
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/alecthomas/units v0.0.0-20210208195552-ff826a37aa15
+	github.com/alicebob/gopher-json v0.0.0-20200520072559-a9ecdc9d1d3a // indirect
```

Comment:
Were you updating some deps too?

Reply:
I didn't. These has been picked up from the Cortex upgrade. If I remove the cortex `replace` directive and re-run `go mod vendor && go mod tidy` the `go.mod` rollbacks to the `master` version.

Reply:
👍🏽